dspace-server-webapp/src/main/java/org/dspace/app/rest/repository/FindableObjectRepository.java

```diff
@@ -0,0 +1,33 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.app.rest.repository;
+
+import java.io.Serializable;
+import java.sql.SQLException;
+
+import org.dspace.core.Context;
+import org.dspace.core.ReloadableEntity;
+
+/**
+ * This interface must be implemented by all the rest repository that deal with resources that can be make
+ * {@link FindableObject}
```

Comment:
This JavaDocs needs updating, as `FindableObject` was removed.

Reply:
done, https://github.com/DSpace/DSpace/pull/2663/files#diff-431cf6863613e58075585b11aa41cb2eR17